### ISSUE TITLE
[Feature Request] Displays and visual stuff.

### ISSUE THREAD

**Issue by zekinh** [Lunat1q/Catchem-PoGo#129]
_**Hello all, so i been using bots for a while now and i think this one might have a lot to give. This are the things i personally think that are needed or could be indeed helpfull.**_

## **Display for each:**
Pokeballs / GreatBalls / Ultraballs
Potions/SuperPotions/MaxPotions

## **>Battle Gyms and auto collect coins<**

This one might not see as important but i would like the bot to actually give me 10 _coin_s every so often (or more).

## More XP per H : 
 I left the bot almost 12 hs and it only gave me (125000xp) , i have to admit that at least i didnt get ban at all (remarcable)

# **Pokemon Infformation:** Attacks individually
The display menu seems a little rusty still, it would be cool to be more detailed as the bot created by (@ar1i) 

Sniper and so its not needed for now (its a ban for sure so lets wait on that, or maybe put the option 95% of ban if using it and lets people decide).

That's it for what **i** think it would be could. Seeya

**Comment by itterror** [comment]
I don't feel battling gyms should be an aspect of botting.

**Comment by Void0Star** [comment]
i agree i think gym battle's should be left out

**Comment by PassiveModding** [comment]
making gym bots would ruin the game

the higher the xp rate the higher the ban chance
